config_management/domains/models.py

```diff
@@ -1,14 +1,22 @@
 from django.db import models
+from django.core.exceptions import ValidationError
 
 # Create your models here.
 
+#validates if the domain name entered is in lower case on not on 'ADMIN PAGE'
+def validate_case_sentitivity(string): 
+	if string.islower()==True:
+		return string
+	else:
+		raise ValidationError("Enter lower case domain name") #raise error in admin page
```

Comment:
We shouldn't ask client to lower case it.

Reply:
validation is done for admin page only as written in comments!